golang/pkg/dagent/dagent.go

```diff
@@ -34,12 +34,19 @@ func Serve(cfg *config.Configuration) {
 
 	update.InitUpdater(cfg)
 
-	grpcParams := grpc.GrpcTokenToConnectionParams(cfg.GrpcToken, cfg.GrpcInsecure)
+	image, err := utils.GetOwnContainerImage(nil)
```

Comment:
Are you tried to run this not inside a docker? 🤔

Reply:
Yes and it used to work untils this part 😅
Working on it.